app/src/androidTest/java/org/pesmypetcare/mypetcare/activities/TestLoginActivity.java

```diff
@@ -0,0 +1,101 @@
+package org.pesmypetcare.mypetcare.activities;
+
+import androidx.test.ext.junit.runners.AndroidJUnit4;
+import androidx.test.filters.LargeTest;
+import androidx.test.rule.ActivityTestRule;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.pesmypetcare.mypetcare.R;
+
+import static androidx.test.espresso.Espresso.onView;
+import static androidx.test.espresso.action.ViewActions.click;
+import static androidx.test.espresso.action.ViewActions.closeSoftKeyboard;
+import static androidx.test.espresso.action.ViewActions.typeText;
+import static androidx.test.espresso.assertion.ViewAssertions.matches;
+import static androidx.test.espresso.matcher.ViewMatchers.isDisplayed;
+import static androidx.test.espresso.matcher.ViewMatchers.withId;
+import static androidx.test.espresso.matcher.ViewMatchers.withText;
+
+@RunWith(AndroidJUnit4.class)
+@LargeTest
+public class TestLoginActivity {
+    public static final String LOG_IN = "LOG IN";
+    @Rule
+    public ActivityTestRule<LoginActivity> activityRule = new ActivityTestRule<>(LoginActivity.class);
+
+    @Test
+    public void shouldMakeTheSignUp() {
+        onView(withId(R.id.signUpUsernameText)).perform(typeText("johnDoe"), closeSoftKeyboard());
+        onView(withId(R.id.signUpMailText)).perform(typeText("johndoe@gmail.com"), closeSoftKeyboard());
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The String "johndoe@gmail.com" appears 4 times in the file.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5239997)